duckduckgo/iOS
# iOS VPN Settings part 1

Task/Issue URL: https://app.asana.com/0/0/1205750157310738/f

**Description**:

Adds the new VPN Settings items + the VPN Settings screen. The next PR (part 2) will include the implementation for the VPN Notifications screen.

**Steps to test this PR**:
1. Make sure you are an internal user (Settings -> Debug -> Internal User). Make sure you close the Settings before proceeding.
2. Navigate to the Network Protection Status screen (Settings -> Network Protection).
3. **Observe the section title Manage with the two new items: VPN Settings & VPN Notifications**
4. Select VPN Settings
5. **Observe the two sections each with a disabled toggle item for: Always On & Secure DNS**
6. Generally check that everything matches [the designs](https://www.figma.com/file/d7d8ZGkWD0W5ExhBI0tESv/O-N---VPN-Settings-Templates?type=design&node-id=527-50263&mode=design&t=Tcd85WK8t225oXHa-0)

**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


—
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
